api/sys_monitor.go

```diff
@@ -20,7 +18,7 @@ func (c *Sys) Monitor(ctx context.Context, logLevel string, logFormat string) (c
 		r.Params.Add("log_level", logLevel)
 	}
 
-	if logFormat == "" || logFormat == logging.UnspecifiedFormat.String() {
+	if logFormat == "" {
```

Comment:
❓ This seems like it will change the behaviour. It looks like the if the logFormat _is_ unspecified, it will no longer be changed to `standard`.

Reply:
Good catch.  Yes, I considered this, but couldn't come up with a legitimate scenario in which that could happen.  Happy to discuss further.

Reply:
No, that's fine. I was looking at this through a microscope, so if you thought about it in the greater context, I'm fine.